File: src/main/java/gr/apt/repository/notification/NotificationRecipientRolesRepository.java
package gr.apt.repository.notification;

import gr.apt.persistence.entity.notification.NotificationRecipientRoles;
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;

import javax.enterprise.context.ApplicationScoped;
import java.math.BigInteger;
@ApplicationScoped
public class NotificationRecipientRolesRepository implements PanacheRepositoryBase<NotificationRecipientRoles, BigInteger> {
}

File: src/main/java/gr/apt/repository/notification/NotificationRecipientRolesRepository.kt
package gr.apt.repository.notification

import gr.apt.persistence.entity.notification.NotificationRecipientRoles
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase
import java.math.BigInteger
import javax.enterprise.context.ApplicationScoped

@ApplicationScoped
class NotificationRecipientRolesRepository : PanacheRepositoryBase<NotificationRecipientRoles?, BigInteger?>